lib/bonny/config.ex

```diff
@@ -122,6 +122,14 @@ defmodule Bonny.Config do
     end
   end
 
+  @doc """
+  Kubernetes APIVersion used. Defaults to `apiextensions.k8s.io/v1`
+  """
+  @spec api_version() :: binary
+  def api_version() do
+    Application.get_env(:bonny, :api_version, "apiextensions.k8s.io/v1")
```

Comment:
If we want to get this into a minor version release in the meantime, let's default it to v1beta1 for backwards compact. Otherwise, if you are open to waiting a few weeks for a major release, we can keep it v1.